torch/types.py

```diff
@@ -12,30 +10,27 @@
     int as _int,
     str as _str,
 )
-from typing import Any, Dict, List, Sequence, Tuple, TYPE_CHECKING, Union
+from collections.abc import Sequence
+from typing import Any, TYPE_CHECKING, Union
 from typing_extensions import TypeAlias
 
 # `as` imports have better static analysis support than assignment `ExposedType: TypeAlias = HiddenType`
-from torch import (  # noqa: F401
+from torch import (
     device as _device,
-    DispatchKey as DispatchKey,
+    DispatchKey,
     dtype as _dtype,
-    layout as _layout,
-    qscheme as _qscheme,
-    Size as Size,
-    SymBool as SymBool,
-    SymFloat as SymFloat,
-    SymInt as SymInt,
-    Tensor as Tensor,
+    Size,
+    SymBool,
+    SymFloat,
+    SymInt,
+    Tensor,
 )
 
 
 if TYPE_CHECKING:
     from torch.autograd.graph import GradientEdge
 
 
-__all__ = ["Number", "Device", "Storage"]
```

Comment:
Why do we move this to the bottom of the file? The `__all__` list should be placed after import statements.

Reply:
Because it is appended with classes below. Follow your advice to move it to the old place.